cmd/selfupdatectl/internal/cloud/aws.go

```diff
@@ -27,10 +29,12 @@ func Exists(path string) bool {
 	return true
 }
 
+// NewAWSSessionFromEnvironment create a new session from environment variable
```

Comment:
Really this one wants a bit more info. For example which environment variables are used and what do they mean